### PR TITLE
Betho playground

### PR DESCRIPTION
**Pull request by iberthony**
# PR Details
- Staking VTX (testnet)
- Balance fetching fix (I hope so)
- Testnet Proposals/Voting
- -Fix verto Connect

### CODE REVIEW

File: app/src/pages/vDexNode/Index.vue [hunk 1]
```diff
@@ -696,7 +698,11 @@ export default {
       script: '',
       now: '',
       eosApi: null,
-      account: {},
+      account: {
+        key: 'EOS8UrDjUkeVxfUzUS1hZQtmaGkdWbGLExyzKF6569kRMR5TzSnQT',
+        privateKey: '5JDCvBSasZRiyHXCkGNQC7EXdTNjima4MXKoYCbs9asRiNvDukc',
```
Comment (gluneau) [inline app/src/pages/vDexNode/Index.vue:703]: Danger, Danger Will Robinson!

File: app/src/components/Verto/StakeStepper.vue [hunk 2]
```diff
@@ -481,14 +497,14 @@ export default {
       this.checkAmount()
     },
     checkAmount () {
-      let stake_per = Math.round((0.01 + (0.001 * this.stakePeriod)) * 1000) / 1000
+      let stake_per = (1 + this.stakePeriod / 10.0) / 100
```
Comment (gluneau) [inline app/src/components/Verto/StakeStepper.vue:500]: I don't think this is right. 
Prove me wrong!

File: app/src/util/Wallets2Tokens.js [hunk 3]
```diff
@@ -215,8 +216,8 @@ class Wallets2Tokens {
 
     // this.$store.state.wallets.portfolioTotal = this.$store.state.wallets.portfolioTotal + usdValue * t.amount
     // console.log('this.$store.state.wallets', this.$store.state)
-    store.commit('wallets/updateTokens', this.tableData)
-    store.commit('wallets/updatePortfolioTotal', store.state.wallets.portfolioTotal)
+    // store.commit('wallets/updateTokens', this.tableData)
+    // store.commit('wallets/updatePortfolioTotal', store.state.wallets.portfolioTotal)
   }
   async getAllAssets () {
     await axios.get('https://volentix.info/get_assets')
```
Comment (gluneau) [inline app/src/util/Wallets2Tokens.js:223]: Interesting....

What is that?  Where is the repo for this?
Where is this server hosted?

File: app/src/components/Verto/StakeStepper.vue (hunk 2 above)
Comment (iberthony) [reply app/src/components/Verto/StakeStepper.vue:500]: I got it from the contract... I saw that the rewards value were not equal to the expected total calculated in the UI

` asset calculate_subsidy(asset quantity, uint16_t periods_num) 
   {
      check_quantity(quantity);
      check(periods_num <= 10, "max allowed periods num is 10 stake periods");
      double subsidy_part = ( 1 + periods_num / 10.0 ) / 100;
      asset subsidy = asset(0, symbol(TOKEN_SYMBOL, SYMBOL_PRE_DIGIT));
      subsidy.amount = quantity.amount * subsidy_part;
      return subsidy; 
   }`

File: app/src/util/Wallets2Tokens.js (hunk 3 above)
Comment (iberthony) [reply app/src/util/Wallets2Tokens.js:223]: Are you referring to getAllAssets() ? It found it there.

I am updating the wallet elsewhere...

**Pull request merged**